freqtrade/optimize/hyperopt.py

```diff
@@ -197,12 +210,20 @@ def log_trials_result(self) -> None:
                 # Also round to 5 digits after the decimal point
                 print(f"Stoploss: {round(params.get('stoploss'), 5)}")
 
+    def is_best(self, results) -> bool:
+        return results['loss'] < self.current_best_loss
+
     def log_results(self, results) -> None:
         """
         Log results if it is better than any previous evaluation
         """
         print_all = self.config.get('print_all', False)
-        is_best_loss = results['loss'] < self.current_best_loss
+        is_best_loss = self.is_best(results)
+
+        if not print_all:
+            print('.', end='' if results['current_epoch'] % 100 != 0 else None)  # type: ignore
```

Comment:
what is the intend of this? 
print a newline every 100 epochs? (just to understand ...) ?

Reply:
It prints a dot and a newline after every 100th epoch and prints a single dot otherwise. (if the result is not a best result).

This is needed to separate the next log message ("Saving N00 epochs.") from the output (from the dots), otherwise it looks messy on the terminal.

Reply:
good point :+1:

Reply:
A comment should be added here, if the intention of this is unclear (for future devs)...